engine/src/main/java/org/hibernate/validator/BaseHibernateValidatorConfiguration.java

```diff
@@ -119,6 +119,16 @@
 	@Incubating
 	String GETTER_PROPERTY_SELECTION_STRATEGY_CLASSNAME = "hibernate.validator.getter_property_selection_strategy";
 
+
+	/**
+	 * Property corresponding to the {@link #ignoreClassLevelOnPropertyFail} method.
+	 * Accepts {@code true} or {@code false}. Defaults to {@code false}.
+	 *
+	 * @since 6.1.0
+	 */
+	@Incubating
+	String IGNORE_CLASS_LEVEL_ON_PROPERTY_FAIL = "hibernate.validator.ignore_class_level_on_property_fail";
```

Comment:
I wonder if we should try to find a naming similar to the fail fast mode as it's really similar to it.

Maybe `fail_fast_on_property_violation`?

It's not easy to find a right name for this sort of configuration knob.

Happy to discuss it further (maybe it will be easier on Zulip).